DEFAULT_CONFIG.json5

```diff
@@ -204,20 +206,77 @@
   //       "key_exprs": [
   //         "test/demo"
   //       ],
+  //     },
+  //     {
+  //       "id": "rule2",
+  //       "actions": [
```

Comment:
I think `actions` is the wrong word here, as those are more message kinds.

Reply:
The naming is from ACL SOTA: subjects (interface, cert, username) perform actions (in the form of messages) on resources (key_exprs).

Reply:
But if you want to set a policy that prevents a node from _receiving_ a Subcriber Declaration that is configured as allowing **`declare_subscriber` actions** for egress flow. Even though the node receiving the declaration isn't doing anything. Same goes for `put` actions.

Reply:
After further discussion within the team, the config's perspective will be changed from `actions` to `messages`. The naming will be updated in this PR.